utils/file.go

```diff
@@ -0,0 +1,18 @@
+package utils
+
+import (
+	"bufio"
+	"os"
+	"strings"
+)
+
+func ReadStringFromFile(f *os.File) string {
```

Comment:
What if we called this `ReadFile` in the root `metro2` project?

Reply:
changed